packages/common-substrate/src/project/types.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {SubqlDatasource} from '@subql/types';
+
+export interface IProjectManifest {
+  specVersion: string;
+  description: string;
+  repository: string;
+  dataSources: SubqlDatasource[];
+  toDeployment(): string;
+  validate(): void;
+}
+
+export interface ProjectNetworkConfig {
```

Comment:
These could live in `@subql/common`. Remember  `genesisHash` should just be an alias